lib/_stream_readable.js

```diff
@@ -18,7 +18,7 @@ var Buffer = require('buffer').Buffer;
 
 Readable.ReadableState = ReadableState;
 
-var EE = require('events').EventEmitter;
+var EE = require('events');
```

Comment:
we do [test it](https://travis-ci.org/nodejs/readable-stream/builds/84955012) against 0.8.x